src/components/home/About.jsx

```diff
@@ -7,24 +7,23 @@ import logo from "@/public/assets/logo.svg";
 
 const About = () => {
   return (
-    <div className="flex flex-col items-center my-[4%]">
-      <div className="flex items-center justify-center px-[10%]">
-        <div className="flex w-2/5 justify-center items-start">
-          <Image src={logo} alt="Picture of logo" className="z-1" />
+    <div className="flex flex-col items-center my-4">
+      <div className="flex flex-col sm:flex-row items-center justify-center px-10">
+        <div className="flex justify-center items-center w-full sm:w-2/5">
+          <Image src={logo} alt="Picture of logo" className="z-10" />
         </div>
 
-        <div className="flex-col text-center w-3/5 mx-auto text-2xl mb-5">
+        <div className="flex flex-col items-center text-center w-full sm:w-3/5 mt-5 sm:mt-0 text-2xl">
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/fa6fab29-4d5a-440f-86e0-b1e60cdef546)

text is too close to the clouds, add a little padding